File: javascript/menor_de_tres.js
const v1 = parseInt(prompt("Digite o primeiro valor"));
const v2 = parseInt(prompt("Digite o segundo valor"));
const v3 = parseInt(prompt("Digite o terceiro valor"));

let menor;

if (v1 <= v2 && v1 <= v3) {
    menor = v1;
} else if (v2 <= v1 && v2 <= v3) {
    menor = v2;
} else {
    menor = v3;
}

console.log("Menor = " + menor);

File: python/menor_de_tres.py
v1 = int(input("Primeiro valor_unitario: "))
v2 = int(input("Segundo valor_unitario: "))
v3 = int(input("Terceiro valor_unitario: "))

menor = v1

if v2 < menor:
    menor = v2
elif v3 < menor:
    menor = v3

print(f"MENOR = {menor}")
